### ISSUE TITLE
DdlParserException if subject contains line breaks

### ISSUE THREAD

**Issue by jovicde** [empira/MigraDoc-1.5#3]
When add line breaks (\r\n) to a document's subject property, a DdlParserException is thrown:

> MigraDoc.DocumentObjectModel.IO.DdlParserException: '']' expected, found '}'.'
> Error	{[(9,3):] xxx DDL1001: ']' expected, found '}'.

```csharp
Document = new Document
            {
                Info =
                {
                    Title = "A Title",
                    Subject = "A subject with a \r\n line break",
                    Author = "Jovic"
                },
             };
```
A stumbled across this issue when I added the WPF DocumentPreview control and set it's Ddl property. The stack trace is:

```
at MigraDoc.DocumentObjectModel.IO.DdlParser.ThrowParserException(DomMsgID errorCode, Object[] parms)
   at MigraDoc.DocumentObjectModel.IO.DdlParser.AssertSymbol(Symbol symbol)
   at MigraDoc.DocumentObjectModel.IO.DdlParser.ParseAttributes(DocumentObject element, Boolean readNextSymbol)
   at MigraDoc.DocumentObjectModel.IO.DdlParser.ParseDocument(Document document)
   at MigraDoc.DocumentObjectModel.IO.DdlReader.ReadDocument()
   at MigraDoc.DocumentObjectModel.IO.DdlReader.DocumentFromString(String ddl)
   at MigraDoc.Rendering.Windows.DocumentPreview.DdlUpdated()
```

**Comment by TH-Soft** [comment]
My first idea: do not use line breaks in document properties.
Adobe Reader DC only shows the part before the line break.

So maybe MigraDoc should throw exceptions when the document properties contain line breaks or other unsupported control characters.

**Comment by jovicde** [comment]
Yes, the workaround is obvious, so it's not a major issue.
The current behaviour is inconsistent in my opinion: The serialization works without errors, but the deserializer stumbles across that line break.

**Comment by ThomasHoevel** [comment]
Problem still exists with PDFsharp 6.0.0. But I close here as we won't fix it for PDFsharp 1.5.
